plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/internal/refresh/listeners/FontFormatUpdater.java

```diff
@@ -133,21 +166,27 @@ public Command transactionAboutToCommit(final ResourceSetChangeEvent event) thro
                                 BasicLabelStyleDescription description = edgeStyle.getBeginLabelStyle().getDescription();
                                 // if format is different between default (Sirius Description/odesign) and GMF
                                 if (!isSameLabelFormat(fontStyle, description.getLabelFormat())) {
-                                    cc.append(AddCommand.create(getTarget(), edgeStyle.getBeginLabelStyle(), customFeatureAttr, labelFormat));
+                                    cc.append(new AddLabelFormatCommand(getTarget(), edgeStyle.getBeginLabelStyle()));
```

Comment:
The above line 157, `EAttribute customFeatureAttr = ViewpointPackage.Literals.CUSTOMIZABLE__CUSTOM_FEATURES;`,  must be deleted.